pubspec.yaml

```diff
@@ -47,10 +47,11 @@ dependencies:
   image_picker:
   geolocator:
   mapbox_gl: ^0.11.0
+  auth_buttons: ^0.0.6
+  http: ^0.12.1
```

Comment:
where are we using http?

Reply:
To convert imageurl to file type we are using http

Reply:
And why are we doing that? Are you caching the file locally and then displaying it?

Reply:
No, when we signin through google , google provide us an imageurl of our profile pic but the current method of storing profile images jt takes an file image type as input and stores it in the storage bucket so i had to convert the google image url into file type for that
Thus i had to use http
But don't worry , since when i'll raise another issue for refactoring images in that i won't be requiring http since i can directly store image url on firebase so that time i would remove the http package @Sameerkash

Reply:
You could also directly store that imageUrl that google signing provides as a value along with name and email in firestore instead of duplicating it in storage.

> But don't worry , since when i'll raise another issue for refactoring images in that i won't be requiring http since i can directly store image url on firebase so that time i would remove the http package @Sameerkash

yeah cool

Reply:
Yeah @Sameerkash  i know that but since the the current method fetches profile image from storage bucket and not from cloudstore 
Since i've i would have to change that all
That's why i said i can fix that in an anothe issue